axelrod/tests/strategies/test_finite_state_machines.py

```diff
@@ -622,17 +643,22 @@ def test_strategy(self):
         self.transitions_test(state_and_actions)
 
         # finished: 0, 5, 10
-        state_and_actions = [(0, D), (12, D), (11, D)] + [(5, D), (10, C), (11, D), (5, D), (10, D), (8, C)] * 3
+        state_and_actions = [(0, D), (12, D), (11, D)] + [(5, D), (10, C),
+                                                          (11, D), (5, D),
+                                                          (10, D), (8, C)] * 3
         self.transitions_test(state_and_actions)
 
         # finished: 0, 2, 5, 10, 11, 12, 15
-        state_and_actions = ([(0, D), (12, C), (8, D), (5, D), (10, C), (11, C), (15, C), (15, C), (15, D)] +
+        state_and_actions = ([(0, D), (12, C), (8, D), (5, D),
+                              (10, C), (11, C), (15, C), (15, C), (15, D)] +
                              [(2, C)] * 3 + [(2, D), (14, C), (13, C)])
         self.transitions_test(state_and_actions)
 
         # finished: 0, 2, 3, 5, 10, 11, 12, 13, 14, 15
         to_state_fourteen = [(0, D), (12, D), (11, C), (15, D), (2, D)]
-        state_and_actions = to_state_fourteen + [(14, D), (13, C), (13, C), (13, D), (7, C)] + [(3, D), (3, C)] * 3
+        state_and_actions = to_state_fourteen + [(14, D), (13, C), (13, C),
+                                                 (13, D), (7, C)] + [(3, D),
+                                                                     (3, C)] * 3
```

Comment:
Yeah, there were loads of 80 character PEP8 rules so I let my auto linter fix it. I can clear it up if you'd prefer?